spec/factories/course_condition_assessments.rb

```diff
@@ -3,7 +3,7 @@
           class: Course::Condition::Assessment.name, aliases: [:assessment_condition] do
     course
     assessment
-    association :conditional, factory: :course_achievement
+    association :conditional, factory: :assessment
```

Comment:
wait wait... this is a conditional. You need a default conditional then add traits to it that will override the association. There should still be traits.